### PR TITLE
Network provider

### PR DESCRIPTION
**Pull request by stgleb**
Add network support on backend

### CODE REVIEW

File: templates/network.sh.tpl [hunk 1]
```diff
@@ -1102,254 +1011,5 @@ sudo kubectl create -f calico.yaml
...
-              hostPath:
-                path: /lib/modules
-            - name: xtables-lock
-              hostPath:
-                path: /run/xtables.lock
-                type: FileOrCreate
-      updateStrategy:
-        type: RollingUpdate
-EOF'
+kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
```
Comment (taaraora) [inline templates/network.sh.tpl]: May be it is better to put all these entities in a separate files? But do not load from external server?
```https://cloud.weave.works``` can be  firewalled.

Comment (egopher) [reply templates/network.sh.tpl]: @stgleb Please copy it there instead of using external server.

Comment (stgleb) [reply templates/network.sh.tpl]: That doesn't work even though content is almost identical, you can try it. I've opened issue to wave github.

Comment (stgleb) [reply templates/network.sh.tpl]: I agree with you, but weave installation is weird and contain some dynamic things inside that can't be just hard coded.  I didn't dig it deeper, but when install from the saved file - it fails.

Comment (stgleb) [reply templates/network.sh.tpl]: Done

**Pull request merged**